DEVELOPING.md

```diff
@@ -51,6 +52,16 @@ For each example there are 3 modes:
 
 - `/examples/abc.html` points to prod. This file would not reflect your local changes.
 
+## Manual visual testing with gemini
```

Comment:
I followed these instructions and I get the following output in the console:

```bash
➜  orion git:(chores/setup-visual-testing-2) ./node_modules/.bin/gemini test gemini/*
Sauce Connect opening tunnel.
Sauce Connect ready.
✘ default button default [chrome]
Could not find element with css selector specified in setCaptureElements: .example
✘ default button hovered [chrome]
Could not find element with css selector in actions chain: orion-button
✘ disabled button default [chrome]
Could not find element with css selector specified in setCaptureElements: .example
✘ disabled button hovered [chrome]
Could not find element with css selector in actions chain: orion-button
✘ small button default [chrome]
Could not find element with css selector specified in setCaptureElements: .example
✘ small button hovered [chrome]
Could not find element with css selector in actions chain: orion-button
✘ large button default [chrome]
Could not find element with css selector specified in setCaptureElements: .example
✘ large button hovered [chrome]
Could not find element with css selector in actions chain: orion-button
Total: 8 Passed: 0 Failed: 8 Skipped: 0 Retries: 0
Closing Sauce Connect.
```

I verified that storybook is running on `localhost:9001`. Here is a link to the test details: https://saucelabs.com/beta/tests/cefeaa8f45eb4c1da586088fee81c439/watch#12

I'm on my home network right now but it's weird that it's not working. I'll also try it at the office tomorrow. Any tips?